src/components/LeafletMarker/index.js

```diff
@@ -32,5 +43,8 @@ export const LMarker = (props) => {
 };
 
 LMarker.propTypes = {
-  device: PropTypes.object
+  device: PropTypes.object,
```

Comment:
`object` As proptype should be avoided. Use `PropTypes.shape({ ... })` instead. That prop type allows for defining its internals whereas `object` can't